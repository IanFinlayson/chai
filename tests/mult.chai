def main():
    print(3 * 6)
    print(1 * 4 * 9)
    print(2.0 * 4)
    print(4 * 0.8)
    print("hi " * 9)
    print(3 * "bye")
    print([1] * 10)
    print(4 * [1, 2, 3])


