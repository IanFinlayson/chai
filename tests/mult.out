18
36
8.0
3.2
hi hi hi hi hi hi hi hi hi 
byebyebye
[1, 1, 1, 1, 1, 1, 1, 1, 1, 1]
[1, 2, 3, 1, 2, 3, 1, 2, 3, 1, 2, 3]
